src/styles/interactiveTokens.js

```diff
@@ -0,0 +1,18 @@
+// Outline: used to highlight the element in focus.
+const focusOutline = (theme) => {
+  return {
+    boxShadow: 0 0 1px 4px theme.colors.intent.focusOutline,
```

Comment:
This needed to be a string, but since it isn't there's a JS error. I've fixed it and will push the fix (along with the button fix and converting this to `rem`) in a few minutes 😄